### ISSUE TITLE
assets se están multiplicando

### ISSUE THREAD

**Issue by Pabl0Parra** [vicenlu/ita-game#43]
Estamos duplicando y triplicando los mismos assets, incluso hay una carpeta de imágenes que contienen algunos duplicados también

**Comment by Bazza6** [comment]
Claramente hay que organizarlos mejor, también he notado que esos svg son demasiado pesados, algunos superan el mb. Creo que primero hay que decidir que formato usar. Si seguir con svg y comprimirlos de alguna forma o pasar a png

**Comment by Spanioulis** [comment]
Sí, mañana concretamos. Es posible que pesen demasiado porque están extraídos directamente de Figma (?); quizá si se descargan de la fuente original pesan menos. Y también cuando estén todos los iconos se podrían añadir a index.js.
